src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--tue calendar--29">
+      <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Используй `nth-child` вместо дополнительного класса